README.md

```diff
@@ -330,6 +331,65 @@ end
 
 5. Your app is now able to access the authenticated `ProductsController` from the `HomeController` using session tokens.
 
+### Marking shops as uninstalled on app/uninstalled webhook
+
+To ensure OAuth still works with session tokens, your app must update its shop records in the event a shop uninstalls your app. You can be notified of these events by subscribing to the `app/uninstall` [webhook](https://shopify.dev/docs/admin-api/rest/reference/events/webhook).
+
+#### Setup app/uninstall webhook
+1. Use the `add_webhook` generator from ShopifyApp to setup the `app/uninstalled` webhook for the app.
```

Comment:
Is this referring to the ShopifyApp cli or the gem? Can we add a link to the source appropriately, for convenience?

Reply:
This is referring to the gem. I just added a link to it